mne/simulation/source.py

```diff
@@ -218,3 +220,132 @@ def simulate_stc(src, labels, stc_data, tmin, tstep, value_fun=None):
 
     stc = SourceEstimate(data, vertices=vertno, tmin=tmin, tstep=tstep)
     return stc
+
+
+def simulate_stc_from_dipole(dipole, bem, info, trans=None, min_dist=5.0,
+                             model_meg=True, model_eeg=True,
+                             n_jobs=1, verbose=None):
+    """Convert dipole object to source estimate and calculate forward operator
+
+    The instance of Dipole is converted to a discrete volume source space,
+    which is then combined with a BEM (can also be a sphere model dict) and
+    the sensor information in info to form a forward operator.
+
+    The source estimate object (with the forward operator) can be projected to
+    sensor-space using :func:`mne.simulation.evoked.simulate_evoked`.
+
+    Parameters
+    ----------
+    dipole : Dipole object
+        Instance of Dipole containing position, orientation and amplitude of
+        one or more dipoles. Several dipoles may occur simultaneously, but
+        when more than one time point is defined, the spacing between them
+        (i.e. the sampling rate) must be constant.
+    bem : str | dict
+        The BEM filename (str) or a loaded sphere model (dict).
+    info : dict
+        Sensor-info etc., e.g., from real evoked file.
+    trans : str | None
+        The head<->MRI transform filename. Must be provided unless BEM
+        is a sphere model.
+    min_dist : float
+        Minimum distance (in milimeters) from the dipole to the inner skull.
+    model_meg : bool
+        Include MEG channels (with or without compensation) in model.
+    model_eeg : bool
+        Include EEG channels in model.
```

Comment:
I don't think we should need these two args. Instead the user can use `pick_info(...)` to pick the correct set of channels to simulate.